rules/coding-style/src/Node/NameImporter.php

```diff
@@ -215,6 +231,35 @@ private function isNonExistingClassLikeAndFunctionFullyQualifiedName(Name $name)
         return ! function_exists($classOrFunctionName);
     }
 
+    private function isFoundInUse(Name $name): bool
+    {
+        $parent = $name->getAttribute(AttributeKey::PARENT_NODE);
+
+        while ($parent) {
```

Comment:
Try getAttribute(AttributeKey::USES) or similar to get those

Reply:
implemented.